emission/tests/netTests/TestWebserver.py

```diff
@@ -67,6 +68,20 @@ def test404Redirect(self):
         self.assertEqual(response.status_code, 301)
         self.assertEqual(response.get_header("Location"), "http://somewhere.else")
 
+    def testResolveAuth(self):
+        import emission.net.api.cfc_webapp as enacw
```

Comment:
given that we are doing a direct import here?

Reply:
This import probably isn't needed. I don't think this test is sensitive to the import and env var that is left unset. I'll try getting rid of it and see what happens.

Reply:
Yeah, this import can be deleted. It isn't doing anything since the import happened globally and there isn't a need to have anything reloaded for this test.